src/content/developers/docs/nodes-and-clients/archive-nodes/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Ethereum Archive Node
+description: 
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+An archive node is an instance of an Ethereum client with configuration to build an archive of all historical states. It is a useful tool for certain use cases but might be more tricky to run than a full node. 
+
+## Prerequisites {#prerequisites}
+
+You should understand concept of [Ethereum node](/developers/docs/nodes-and-clients/), [its architecture](/developers/docs/nodes-and-clients/node-architecture/). [sync strategies](https://ethereum.org/en/developers/docs/nodes-and-clients/#sync-modes).   practices of [running](/developers/docs/nodes-and-clients/run-a-node/) and [using them](/developers/docs/apis/json-rpc/).
+
+## What is an archive node
+
+To grasp the importance of an archive node, let's clarify the concept of "state."
+Ethereum can be referred to as *transaction-based state machine*. It consists of accounts and applications executing transactions which are changing their state. The global data with information about each account and contract is stored in trie database called state. This is handled by the execution layer (EL) client and includes:
+* Account balances and nonces 
+* Contract code and storage
+* Consensus-related data, e.g. Staking Deposit Contract
+
+To interact with the network, verify and produce new blocks, Ethereum clients have to keep up with chain tip and therfore the current state. EL client configured as a full node verifies and follows the latest state of the network. A full node temporarily caches only past few states, e.g. for last 128 blocks, for case of a chain reorg. But the recent state is all what client needs to verify incoming transactions and use the network.  
```

Comment:
```suggestion
To interact with the network, verify and produce new blocks, Ethereum clients have to keep up with the most recent changes (the tip of the chain) and therefore the current state. An EL client configured as a full node verifies and follows the latest state of the network but only caches the past few states, e.g. the state associated with the last 128 blocks, so it can handle chain reorgs and provide fast access to recent data. The recent state is all what client needs to verify incoming transactions and use the network.  
```